packages/strapi-plugin-users-permissions/services/Providers.js

```diff
@@ -172,6 +172,40 @@ const getProfile = async (provider, query, callback) => {
       });
       break;
     }
+    case 'microsoft': {
+      const microsoft = new Purest({
+        provider: 'microsoft',
+        config:{
+          'microsoft': {
+            'https://graph.microsoft.com': {
+              '__domain': {
+                'auth': {
+                  'auth': {'bearer': '[0]'}
+                }
+              },
+              '[version]/{endpoint}': {
+                '__path': {
+                  'alias': '__default',
+                  'version': 'v1.0'
+                }
+              }
+            }
+          }
+        }
+      });
```

Comment:
@derrickmehaffy Here's the part I had to add for Microsoft. You'll most likely need to do the same for Discord.

Reply:
Thanks @synth3tk I'll jump on this :+1: